app/views/pages/index.html.erb

```diff
@@ -16,7 +16,6 @@
   </li>
 </ul>
 <hr/>
-<div id="content"></div>
-
-
-
+<div id="content">
+  <%= react_component('App') %>
```

Comment:
OK -- we should either comment or explicitly say that server rendering is off.

I think the next PR should be two have 2 tabs that show with and without server rendering.

@alexfedoseev @samnang @skv-headless @mapreal19 @PlasticLizard thoughts?